pyecharts/charts/basic_charts/radar.py

```diff
@@ -25,9 +26,18 @@ def add_schema(
         splitarea_opt: Union[opts.SplitAreaOpts, dict] = opts.SplitAreaOpts(),
         axisline_opt: Union[opts.AxisLineOpts, dict] = opts.AxisLineOpts(),
     ):
+        if isinstance(textstyle_opts, opts.TextStyleOpts):

```

Comment:
I think, no long this is needed. https://github.com/pyecharts/pyecharts/pull/1058 has removed such a need!